docs/outlook/debug-autolaunch.md

```diff
@@ -0,0 +1,136 @@
+
+    [!include[Developer registry key](../includes/developer-registry-key.md)]
+
+1. Start Outlook desktop (or restart Outlook if it's already open).
+1. Compose a new message or appointment. You should see the following dialog. Do *not* interact with the dialog yet.
+
+    ![Screenshot of Debug Event-based handler dialog](../images/outlook-win-autolaunch-debug-dialog.png)
+
+## Configure Visual Studio Code
+
+### yo office
+
+1. Back in the command line window, open VS Code.
+
+    ```command&nbsp;line
+    code .
+    ```
+
+1. In VS Code, open the file **./.vscode/launch.json** and add the following excerpt to your list of configurations. Save your changes.
+
+    ```json
+    {
+      "name": "Direct Debugging",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "protocol": "inspector",
+      "timeout": 600000,
+      "trace": true
+    }
+    ```
+
+### Other
+
+1. Create a new folder called **Debugging** (perhaps in your **Desktop** folder).
+1. Open VS Code.
+1. Go to **File** > **Open Folder**, navigate to the folder you just created, then choose **Select Folder**.
+1. Select the **Debug** item in the left menu.
+
+    ![Screenshot of Debug icon in left menu of VS Code](../images/vs-code-debug.png)
+
+1. Select the **create a launch.json file** link.
+
+    ![Screenshot of link to create a launch.json file in VS Code](../images/vs-code-create-launch.json.png)
+
+1. In the menu, select **Edge: Launch** to create a launch.json file.
+1. Add the following excerpt to your list of configurations. Save your changes.
+
+    ```json
+    {
+      "name": "Direct Debugging",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "protocol": "inspector",
+      "timeout": 600000,
+      "trace": true
+    }
+    ```
+
+## Attach VS Code
+
+1. To find the add-in's **bundle.js**, open the following folder in Windows Explorer and search for your add-in's **Id** (found in the manifest).
+
+    ```text
+    %LOCALAPPDATA%\Microsoft\Office\16.0\Wef
+    ```
+
+    Open the folder prefixed with this ID and copy its full path. In VS Code, open **bundle.js** from that folder. The pattern of the file path should be as follows:
+
+    `%LOCALAPPDATA%\Microsoft\Office\16.0\Wef\{[Outlook profile GUID]}\[encoding]\Javascript\[Add-in ID]_[Add-in Version]_[locale]\bundle.js`
+
+1. Place breakpoints in bundle.js where you want the debugger to stop.
+1. In the **DEBUG** dropdown, select the name **Direct Debugging** then select the **Run** button.
+
+    ![Screenshot of selecting Direct Debugging from configuration options in the VS Code Debug dropdown](../images/outlook-win-autolaunch-debug-vsc.png)
+
+## Debug
+
+1. After confirming that the debugger is attached, return to Outlook then choose the **OK** button in the **Debug Event-based handler** dialog.
+
+1. Your breakpoints in VS Code should now be hit, enabling you to debug your event-based activation code.
+
+## Stop debugging
+
+To stop debugging for:
```

Comment:
```suggestion
```
I don't think this format will "read" clean using Narrator. Suggest adding the prepositional phrase to the beginning of each sentence where it applies.

Reply:
Instead of using bullets in the subsequent list, would it help to use numbers instead? My understanding was that starting a bunch of items with the same text is difficult cognitively for some folks.

Reply:
There are only two items, this should be fine even for those who suffer from dyslexia. You can also consider removing "To stop debugging" on the second scenario and simply say "For subsequent Outlook session, to stop debugging... " or just "For subsequent Outlook sessions, ..."